mempool/policy.go

```diff
@@ -321,8 +321,8 @@ func checkTransactionStandard(tx *bchutil.Tx, height int32,
 
 	// A standard transaction must not have more than one output script that
 	// only carries data.
-	if numNullDataOutputs > 1 {
-		str := "more than one transaction output in a nulldata script"
+	if dataCarrierSize > maxDataCarrierSize {
+		str := "transaction nulldata exceeds 223 bytes"
```

Comment:
Shouldn't this use maxDataCarrierSize? That way the message changes when we update the size later. ;)

Reply:
Or just use `txscript.MaxDataCarrierSize`

Reply:
updated.

Reply:
I meant in the str line!

Reply:
Got it, this is fixed now too.